vsintegration/update-vsintegration.cmd

```diff
@@ -117,12 +117,12 @@ echo.
 echo For Release builds:
 echo.
 echo ^> VSIXInstaller.exe /u:"VisualFSharp"
-echo ^> VSIXInstaller.exe release\net40\bin\VisualFSharpOpenSource.vsix
+echo ^> VSIXInstaller.exe release\net40\bin\VisualFSharpFull.vsix
 echo.
 echo For Debug builds:
 echo.
 echo ^> VSIXInstaller.exe /u:"VisualFSharp"
-echo ^> VSIXInstaller.exe debug\net40\bin\VisualFSharpOpenSource.vsix
+echo ^> VSIXInstaller.exe debug\net40\bin\VisualFSharpFull.vsix
```

Comment:
Maybe just change the name of the package to VisualFSharp.vsix

Reply:
Eventually I will, but we have a bunch of setup stuff that's dependent upon the name `VisualFSharpFull` that I didn't want to do in this PR.